dallinger/recruiters.py

```diff
@@ -381,6 +487,68 @@ def _handle_exit_form_submission(self, assignment_id: str, participant_id: str):
         q = _get_queue()
         q.enqueue(worker_function, "AssignmentSubmitted", assignment_id, participant_id)
 
+    def load_service(self, sandbox):
+        return _prolific_service_from_config(sandbox)
+
+    def clean_qualifications(self, experiment_details):
+        requirements = []
+        for requirement in experiment_details["eligibility_requirements"]:
```

Comment:
Once you have refactored into functions as described below it would be more Pythonic to use a list comprehension

Reply:
I refactored both `cleaned_attributes` and the `for requirement in experiment_details["eligibility_requirements"]` into list comprehension